NumberBaseball/NumberBaseball/main.swift

```diff
@@ -1,10 +1,52 @@
 //
 //  NumberBaseball - main.swift
-//  Created by yagom. 
+//  Created by yagom.
```

Comment:
프로젝트 진행한 팀원들의 이름으로 수정하시면 좋을 것 같네요!
```suggestion
//  Created by Etialmoon, Mary.
```